jujugui/static/gui/src/app/components/deployment/input/_input.scss

```diff
@@ -12,8 +12,6 @@
         width: calc(100% - 20px);
         height: 20px;
         margin-bottom: 20px;
-        padding-top: 25px;
-        padding-bottom: 7px;
```

Comment:
You'll need to add the styles somewhere else or apply a `--without-label` class that removes these styles or something as currently with this change the inputs in the deployment flow are broken.